packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -195,12 +195,16 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
   late T currentGame;
 
   Future<void> get loaderFuture => _loaderFuture ??= (() async {
-        assert(currentGame.hasLayout);
-        final onLoad = currentGame.onLoadFuture;
+        final game = currentGame;
+        assert(game.hasLayout);
+        final onLoad = game.onLoadFuture;
         if (onLoad != null) {
           await onLoad;
         }
-        currentGame.onMount();
+        game.onMount();
```

Comment:
should this be `game.mount()` now?

Reply:
It should be indeed, why are the tests passing...

Reply:
Since the other one was still there of course, fixed now.